### ISSUE TITLE
Polygon created from closed LinearRing is not .is_closed ?

### ISSUE THREAD

**Issue by fbpyr** [shapely/shapely#1246]
First of thank you for this amazing library - I find it extremely useful!! 🙂 

## Expected behavior and actual behavior.

`Polygon` created from closed `LinearRing` is not `.is_closed` ?

Maybe I misunderstood something, but I expected the 
Polygon created from a closed LinearRing to also be closed.
Or is there an additional step, to have the Polygon check on 
itself and have it set is_closed to True if appropriate?

## Steps to reproduce the problem.

```python
lring = LinearRing([Point(0,0,0), Point(2,0,0), Point(2,2,0) , Point(0,2,0)  , Point(0,0,0)])
lring.is_closed # True

poly = Polygon(lring)
poly.is_closed # False
```

## Operating system

Win10

## Shapely version and provenance

shapely 1.8.0 from PyPi
```python
shapely.__version__ # '1.8.0'
```

**Comment by Jeremiah-England** [comment]
Based on the current implementation in `main`, this looks very intentional. The docstring for the predicate says "this function will return false for non-linestrings". And the definition of the `is_closed` method returns `True` for LinearRings but everything else is just passed to that predicate (links below).

There is a good discussion on this issue with links to other discussions here: https://github.com/pygeos/pygeos/issues/337. If you want my opinion, I think it makes sense for all Polygons to be closed, just like LinearRings. But I guess it's a matter of definition. From the PostGIS docs page linked in the issue above, it seems that even Points are closed there. It would be interesting to survey of what all simple feature libraries do sometime and see if there is anything like a consensus.

https://github.com/shapely/shapely/blob/d2da26aea0a2d49ea37acca4e9edc111f686aabe/shapely/predicates.py#L108-L130

https://github.com/shapely/shapely/blob/d2da26aea0a2d49ea37acca4e9edc111f686aabe/shapely/geometry/base.py#L505-L512

**Comment by sgillies** [comment]
@fbpyr @Jeremiah-England In version 1.8.2, Polygons and MultiPolygons will be closed by definition, which indeed seems the least surprising result.

**Comment by fbpyr** [comment]
@Jeremiah-England @sgillies thank you so much for your replies! 🙂 
Looking forward to 1.8.2 🎉

**Comment by jonimke** [comment]
@sgillies I noticed that in version 2.0.1 Polygons and MultiPolygons are not closed by definition anymore. Could this defintion be transferred to version 2 as well?

**Comment by nickanthony-dgl** [comment]
This issue appears to be happening again in v2.1.1

**Comment by mwtoews** [comment]
It seems that the fix (7cb0bf03be9e4777b0ba8f2008ea1d83ff2c7716) was applied only for 1.8.2. A similar fix should also be applied to `main`.
